### PR TITLE
Remove integtest.sh in every repo.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@dblock.org>

### Description

A bit of meta work to make PRs in all repos that remove a file needed by https://github.com/opensearch-project/opensearch-build/issues/497.
  
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: META.md [hunk 1]
```diff
@@ -67,3 +68,30 @@ Create a file for the issue body, e.g. `issue.md`.
...
+
+Remove a file, e.g. `integtest.sh`, commit and push.
+
+```
+meta exec "git rm integtest.sh"
+meta exec "git add --all"
+meta exec "git checkout -b remove-integtest-sh"
+meta exec "git commit -s -m 'Removed integtest.sh.'"
+meta exec "git push username remove-integtest-sh"
+meta exec "gh pr create --title 'Removing default integtest.sh.' --body='Coming from https://github.com/opensearch-project/opensearch-build/issues/497, removing default integtest.sh.'"
```
Comment (setiah) [inline META.md:96]: Looks nice! Curious - If a repo doesn't have the integtest.sh file, is that gracefully handled with this?

Comment (dblock) [reply META.md:96]: Yeah, it just fails through. If there's nothing to PR, it doesn't do anything.

**Comment by dblock** [comment]
Lychee fixed in https://github.com/opensearch-project/opensearch-plugins/pull/81

**Review by saratvemulapalli — APPROVED** [review]
Love this!

**Pull request merged**